components/services/bat_ledger/bat_ledger_impl.cc

```diff
@@ -256,14 +257,24 @@ void BatLedgerImpl::OnFetchGrants(
 
 void BatLedgerImpl::FetchGrants(const std::string& lang,
     const std::string& payment_id,
+    const std::string& result_string,
     FetchGrantsCallback callback) {
   // deleted in OnFetchGrants
   auto* holder = new CallbackHolder<FetchGrantsCallback>(
       AsWeakPtr(), std::move(callback));
-  ledger_->FetchGrants(lang, payment_id,
+  ledger_->FetchGrants(lang, payment_id, result_string,
```

Comment:
do we need the `_string` notation for `result_string`, can we change to `result` or `response`?

Reply:
If we are going that way, I would like to see a document that describes all that requirements for the code. Put it in the repo, put it in the wiki.

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (5)